### PR TITLE
Sad path testing

### LINKED ISSUE #18: Sad path testing for search results
Sad path tests for incomplete or invalid params sent to BE. Two tests have been started within `spec/requests/api/v1/search/index_spec.rb` that are currently skipped. Example response of error message in BE README.

- [ ] Happy/sad path testing

- [ ] Update to README (FE screenshots, BE API endpoint documented)

### PR DESCRIPTION
**Pull request by stillsheryl**
Add sad path tests for both the facade and service. Improved the user experience by returning the exact reason for the error.

- [x] Updated Files: search index spec, trip facade spec

- [x] Repo Representation: Back End

- [x] Sad Path Tested

- [x] Happy Path Tested

- [x] All Tests are passing

- [x] Collaborators: Nick, Sheryl

closes #18

### CODE REVIEW

File: app/controllers/api/v1/search_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def index
     trips = TripFacade.get_trips(flight_params)
     if trips.class == Hash
       payload = {
-          error: "Invalid Data",
+          error: trips[:message].first[:errors].first,
```
Comment (dunlapww) [inline app/controllers/api/v1/search_controller.rb:6]: Nice! I like this

**Review by dunlapww — COMMENTED** [review]
I like the clearer error messaging.  Looks like we just need to figure out why its not passing Travis before approval

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/controllers/api/v1/search_controller.rb (hunk 1 above)
Comment (JesseMellinger) [reply app/controllers/api/v1/search_controller.rb:6]: I like the use of specific error messages returned by Kiwi here!

**Review by JesseMellinger — APPROVED** [review]
Sad path tests look good!

**Review by cpfergus1 — APPROVED** [review]

**Pull request merged**